dtc-systems/config/matty-followpath.json

```diff
@@ -0,0 +1,74 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "driver": "osgar.followpath:FollowPath",
+          "in": ["emergency_stop", "pose2d", "path"],
+          "out": ["desired_speed"],
+          "init": {
+            "max_speed": 0.25,
+            "obstacle_stop_dist": 0.4,
+            "path": [[0, 0], [1, 0], [1, 1], [2, 1]],
+            "timeout": 20
+          }
+      },
+      "platform": {
+        "driver": "osgar.platforms.matty:Matty",
+        "in": ["esp_data"],
+        "out": ["esp_data"],
```

Comment:
Maybe more notes?

Reply:
do you mean that in reality there are more inputs/outputs